tests/PHPStan/Rules/Properties/TypesAssignedToPropertiesRuleTest.php

```diff
@@ -140,15 +140,7 @@ public function testTypesAssignedToPropertiesExpressionNames(): void
 				66,
 			],
 			[
-				'Property PropertiesFromArrayIntoObject\Foo::$float_test (float) does not accept float|int|string.',
-				69,
-			],
-			[
-				'Property PropertiesFromArrayIntoObject\Foo::$foo (string) does not accept float|int|string.',
-				69,
-			],
-			[
-				'Property PropertiesFromArrayIntoObject\Foo::$lall (int) does not accept float|int|string.',
+				'Property PropertiesFromArrayIntoObject\Foo::$lall (int) does not accept string.',
```

Comment:
This change makes sense to me. The error must be same with line 54.
https://github.com/phpstan/phpstan-src/blob/d4b11635bed572788a323cbaec7032b8d9957e07/tests/PHPStan/Rules/Properties/data/properties-from-array-into-object.php#L48-L70